src/ontology/cl-edit.owl

```diff
@@ -32727,6 +32728,15 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
 AnnotationAssertion(rdfs:label obo:CL_4052011 "follicle associated enterocyte")
 SubClassOf(obo:CL_4052011 obo:CL_0000584)
 
+# Class: obo:CL_4052017 (choroid plexus capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18480313") Annotation(oboInOwl:hasDbXref "PMID:32375819") Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK27998/") obo:IAO_0000115 obo:CL_4052017 "A capillary endothelial cell that is part of the choroid plexus stroma, characterized by its fenestrated nature with 60 to 80 nm fenestrations and lack of tight junctions. This fenestrated structure allows for the rapid delivery of water and other components, aiding in the production of cerebrospinal fluid (CSF).")
+AnnotationAssertion(terms:contributor obo:CL_4052017 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052017 "2024-10-01T11:38:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4052017 "choroid plexus capillary endothelial cell")
+EquivalentClasses(obo:CL_4052017 ObjectIntersectionOf(obo:CL_0002144 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005206)))
```

Comment:
should be part of 'choroid plexus', not 'choroid plexus stroma' which is defined as "Stromal matrix _surrounding_ blood vessels within the choroid plexus"